src/oemof/solph/processing.py

```diff
@@ -251,7 +253,15 @@ def __separate_attrs(system, get_flows=False, exclude_none=True):
     {(node1, node2): {'scalars': {'attr1': scalar, 'attr2': 'text'},
     'sequences': {'attr1': iterable, 'attr2': iterable}}}
 
-    om : A solved oemof.solph.Model.
+    system:
+        A solved oemof.solph.Model or oemof.solph.Energysystem
+    exclude_attrs: List[str]
+        Optional list of additional attributes which shall be excluded from
+        parameter dict
```

Comment:
The documentation states that the argument is optional, but as no default (`None`?) is given, it is obligatory.

Reply:
You are right. I moved initialization of `exclude_attrs` into function `parameter_as_dict` where it is marked correctly as optional and forgot about init in `separate_attrs`. Will add it there too. Thanks